File: 3st/Hyeonseo/programmer/문자 반복 출력하기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, int n) {
    string answer = "";
    for(int i = 0; i < my_string.length();i++) {
        for(int j = 0; j < n; j++) {
            answer += my_string[i];
        }
    }
    return answer;
}

File: 3st/Hyeonseo/programmer/문자 반복 출력하기.java
import java.util.*;

class Solution {
    public String solution(String my_string, int n) {
        String answer = "";
        char c[] = my_string.toCharArray();
        for(int i = 0; i < c.length; i++) {
            for(int j = 0; j < n; j++) {
                answer += c[i];
            }
        }
        return answer;
    }
}